examples/gno.land/r/matijamarjanovic/home/home.gno

```diff
@@ -0,0 +1,230 @@
+}
+
+func VoteRed() {
+
+	ugnotAmount := std.GetOrigSend().AmountOf("ugnot")
+
+	redVotes = redVotes + ugnotAmount/750
+	red, green, blue := calculateRGB(redVotes, greenVotes, blueVotes)
+	currentColor = ufmt.Sprintf(`rgb(%s, %s, %s)`, strconv.Itoa(red), strconv.Itoa(green), strconv.Itoa(blue))
+}
+
+func VoteGreen() {
+	ugnotAmount := std.GetOrigSend().AmountOf("ugnot")
+
+	greenVotes = greenVotes + ugnotAmount/750
+	red, green, blue := calculateRGB(redVotes, greenVotes, blueVotes)
+	currentColor = ufmt.Sprintf(`rgb(%s, %s, %s)`, strconv.Itoa(red), strconv.Itoa(green), strconv.Itoa(blue))
+}
+
+func VoteBlue() {
+	ugnotAmount := std.GetOrigSend().AmountOf("ugnot")
+
+	blueVotes = blueVotes + ugnotAmount/750
+	red, green, blue := calculateRGB(redVotes, greenVotes, blueVotes)
+	currentColor = ufmt.Sprintf(`rgb(%s, %s, %s)`, strconv.Itoa(red), strconv.Itoa(green), strconv.Itoa(blue))
+
+}
+
+func CollectBalance() {
+	AssertAuthorized()
+
+	banker := std.GetBanker(std.BankerTypeRealmSend)
+	ownerAddr := Address()
+
+	banker.SendCoins(std.CurrentRealm().Addr(), ownerAddr, banker.GetCoins(std.CurrentRealm().Addr()))
+}
+
+func Render(path string) string {
+
+	out := ufmt.Sprintf("<div style='font-family: \"Trebuchet MS\", sans-serif;font-size: 5rem; color: %s;'>\n\n", currentColor)
+	out += ufmt.Sprintf("# Matija's Homepage \n\n%s", "")
+	out += "</div>\n\n"
+	out += "<div style='font-family: \"Trebuchet MS\", sans-serif; text-align: justify;'>\n\n"
+	out += renderAboutMe()
+	out += "\n\n"
+	out += ufmt.Sprintf("<h3 style='color: %s;'>Vote for the color of the page - the more you pay the more effect it will have on the homepage ;)</h3>\n", currentColor)
+	out += renderButtons()
+	out += "</div>"
+	out += renderGitHubProjects()
+	out += "\n\n"
+
+	return out
+}
+
+func renderAboutMe() string {
+	out := "<div class='columns-2'>"
+
+	out += "<div>\n\n"
+	out += ufmt.Sprintf(
+		`<img src='%s' alt='Profile Picture' style='border-radius: 50%%; width: 200px; transition: transform 0.3s ease; outline: 5px solid %s;' />`,
+		pfp, currentColor)
+	out += ufmt.Sprintf("<p style='color: %s'>%s</p>\n\n", currentColor, pfpCaption)
+	out += "</div>\n\n"
+
+	out += ufmt.Sprintf("<div style='color: %s'>\n\n", currentColor)
+	out += abtMe + "\n\n"
+	out += "</div>\n\n"
+
+	out += "</div>\n\n"
+
+	return out
+}
+
+func renderGitHubProjects() string {
+	out := `<div style="display: flex; justify-content: center; gap: 10px;">`
+
+	// GitHub Button
+	out += `<a href="https://github.com/matijamarjanovic" target="_blank" style="text-decoration: none;">`
+	out += `<button style="background-color: black; color: white; padding: 8px 16px; border: none; border-radius: 5px; cursor: pointer; display: flex; align-items: center; font-size: 14px;">`
+	out += `<img src="https://upload.wikimedia.org/wikipedia/commons/9/91/Octicons-mark-github.svg" alt="GitHub Logo" style="width: 16px; height: 16px; margin-right: 8px;">`
+	out += `GitHub`
+	out += `</button>`
+	out += `</a>`
```

Comment:
I made it this way because I thought it's more readable, besides, Leon mentioned that there is a package that will be used to build the Render() function instead of using raw HTML, so in time this will be fixed if needed.